release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -257,14 +268,14 @@ For more information, see xref:../machine_management/control_plane_machine_manag
 [id="ocp-4-13-machine-config-operator-layering"]
 ==== Support for adding third party and custom content to {op-system}
 
-You can now use {op-system-first} image layering to add {op-system-base-full} and third-party packages to cluster nodes. 
+You can now use {op-system-first} image layering to add {op-system-base-full} and third-party packages to cluster nodes.
 
 For more information, see  xref:../post_installation_configuration/coreos-layering.adoc#coreos-layering[{op-system-first} image layering].
 
 [id="ocp-4-13-machine-config-operator-core"]
 ==== Support for setting the `core` user password
 
-You can now create a password for the {op-system} `core` user. In the event that you cannot use SSH or the `oc debug node` command to access a node, this password allows you to use the `core` user to access the node through a cloud provider serial console or a bare metal baseboard controller manager (BMC). 
+You can now create a password for the {op-system} `core` user. In the event that you cannot use SSH or the `oc debug node` command to access a node, this password allows you to use the `core` user to access the node through a cloud provider serial console or a bare metal baseboard controller manager (BMC).
```

Comment:
```suggestion
You can now create a password for the {op-system} `core` user. If you cannot use SSH or the `oc debug node` command to access a node, this password allows you to use the `core` user to access the node through a cloud provider serial console or a bare metal baseboard controller manager (BMC).
```
See the IBMSG: https://www.ibm.com/docs/en/ibm-style?topic=word-usage#i

Reply:
I will update but to be honest this is not content I created.